tests/Integration/Behaviour/Features/Scenario/CLDR/cldr.feature

```diff
@@ -1,5 +1,6 @@
 # ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cldr
 @reset-database-before-feature
+@clear-cache-after-feature
```

Comment:
Why did you need to change this? Did it impact the product tests?
I'm always very careful with tests related to CLDR.. I don't know why 😅

Reply:
for a record, 
It used to cache languages in memory, so my product multilang scenarios failed. 
e.g. I was expecting only EN and FR langs, but there was multiple other languages created, so values didn't match.